doc/nvim-treesitter.txt

```diff
@@ -0,0 +1,90 @@
+*nvim-treesitter* 
+
+Minimum version of neovim: nightly
+
+Authors: Yazdani Kiyan <yazdani.kiyan@protonmail.com>, Vigouroux Thomas <@EMAIL?>
+
+==============================================================================
+INTRODUCTION                                       *nvim-treesitter-intro*
+
+nvim-treesitter wraps the neovim treesitter api to provide functionnalities such
+as highlighting and incremental selection, and a command to easily install parsers.
+
```

Comment:
Something like this should be added.
```suggestion
This plugin also provides a some utility function to abstract out treesitter so that plugin are easily writable.

```

Reply:
maybe we should add this when it is available